security/deprecated-blowfish/deprecated-blowfish.yml

```diff
@@ -0,0 +1,12 @@
+- id: deprecated-blowfish
+  name: Inadequate Encryption Strength
+  severity: error
+  message: Blowfish is considered deprecated due to its small block size, AES is the recommended cipher
+  properties:
+    precision: medium
+    tags:
+      - security
+      - vulnerability
+  patterns-either:
+    - pattern: "(Cipher/getInstance #\"Blowfish/.*/.*\")"
```

Comment:
if the code uses `javax.crypto.Cipher` directly it will not match.

Reply:
That's a good point, I added a new pattern with the full package name. This should be sufficient since AFAIK Java imports don´t support aliases.